server/itx/heartbeat.go

```diff
@@ -111,6 +114,10 @@ func (h *HeartbeatHandler) Log() {
 			// RollDpos Concensus Metrics
 			consensusMetrics, err = rolldpos.Metrics()
 			if err != nil {
+				if height == 0 && errors.Cause(err) == statedb.ErrStateNotExist {
+					log.L().Warn("consensus metrics are not ready yet, candidates are empty")
+					return
```

Comment:
is it intended to return error in this case?

Reply:
Oh no, I think we don't have to return in here